apps/accelerate/open_alpha_tensor/open_alpha_tensor/core/actors/stage.py

```diff
@@ -65,17 +76,14 @@ def extract_children_states_from_actions(
     actions = actions.reshape(bs, k, n_steps * len_token)
     vec_dim = state.shape[2]
     u = actions[:, :, :vec_dim].reshape(bs, k, vec_dim, 1, 1)
-    v = actions[:, :, vec_dim : 2 * vec_dim].reshape(  # noqa E203
+    v = actions[:, :, vec_dim : 2 * vec_dim].reshape(
         bs, k, 1, vec_dim, 1
-    )
+    )  # noqa E203
     w = actions[:, :, 2 * vec_dim :].reshape(bs, k, 1, 1, vec_dim)  # noqa E203
     reducing_tensor = u * v * w
-    (
-        reducing_tensor,
-        old_idx_to_new_idx,
-        repetition_map,
-        not_duplicate_indexes,
-    ) = remove_duplicates(reducing_tensor)
+    reducing_tensor, repetition_map, duplicate_indexes = remove_duplicates(
+        reducing_tensor
+    )
```

Comment:
please keep the old code modification (the code in red should replace the code in green)